### PR TITLE
Add docs for how to configure kubectl

### PR DESCRIPTION
**Pull request by ekerfelt**

### CODE REVIEW

File: docs/menu.cue [hunk 1]
```diff
@@ -384,6 +384,11 @@
 									text: "Use Firebase Authentication"
 									path: "/how-to/firebase-auth"
 									file: "how-to/firebase-auth"
+								}, {
```
Comment (eandre) [inline docs/menu.cue:387]: Should we make this a sub-menu of the Kubernetes docs instead?

Comment (ekerfelt) [reply docs/menu.cue:387]: Another level under https://encore.dev/docs/deploy/kubernetes?

Comment (eandre) [reply docs/menu.cue:387]: Yeah exactly

File: docs/menu.cue [hunk 2]
```diff
@@ -468,6 +468,12 @@
 						text: "Kubernetes deployment"
 						path: "/deploy/kubernetes"
 						file: "deploy/kubernetes"
+						inline_menu: [{
+							kind: "basic"
+							text: "Configure kubectl"
+							path: "/deploy/configure-kubectl"
```
Comment (eandre) [inline docs/menu.cue:474]: Should we make this `/deploy/kubernetes/kubectl`?

**Review by DomBlack — APPROVED** [review]

**Pull request merged**